src/flow_solver/flow_solver_cases/cube_flow_uniform_grid.h

```diff
@@ -0,0 +1,43 @@
+#ifndef __CUBE_FLOW_UNIFORM_GRID__
+#define __CUBE_FLOW_UNIFORM_GRID__
+
+#include "flow_solver_case_base.h"
+#include "dg/dg_base.hpp"
+
+namespace PHiLiP {
+namespace FlowSolver {
+
+template <int dim, int nstate>
+class CubeFlow_UniformGrid : public FlowSolverCaseBase<dim, nstate>
+{
+#if PHILIP_DIM==1
+     using Triangulation = dealii::Triangulation<PHILIP_DIM>;
+ #else
+     using Triangulation = dealii::parallel::distributed::Triangulation<PHILIP_DIM>;
+ #endif
+
+ public:
+     explicit CubeFlow_UniformGrid(const Parameters::AllParameters *const parameters_input);
+     
+    /// Function to compute the adaptive time step
+    double get_adaptive_time_step(std::shared_ptr<DGBase<dim,double>> dg) const override;
+
+    /// Function to compute the initial adaptive time step
+    double get_adaptive_time_step_initial(std::shared_ptr<DGBase<dim,double>> dg) override;
+
+    /// Updates the maximum local wave speed
+    void update_maximum_local_wave_speed(DGBase<dim, double> &dg);
+ 
+ private:
```

Comment:
change to protected, otherwise any derived classes cant access these members and they are currently being used in `PeriodicTurbulence`; are the TGV tests passing even with this change?

Reply:
I'm not sure how, but all the TGV cases were passing with that change.
I've made the fix now! :smile: